workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/components/TicketPreview.tsx

```diff
@@ -0,0 +1,184 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import ClayLoadingIndicator from '@clayui/loading-indicator';
+import ClayPanel from '@clayui/panel';
+import {useDraggable} from '@dnd-kit/core';
+import {CSS} from '@dnd-kit/utilities';
+import {useContext, useMemo, useState} from 'react';
+import {QueryClient, useMutation} from 'react-query';
+
+import {TicketsAppContext} from '../context';
+import {Liferay} from '../services/liferay';
+import {assignTicketToMe} from '../services/tickets';
+import {Ticket} from '../types';
+
+const TicketPreview: React.FC<{ticket: Ticket}> = ({ticket}) => {
```

Comment:
Is TicketCard maybe a better name for the whole component? The expanding portion is like a preview, but that seems like only part of the functionality, right?

Reply:
If there is a ticket preview, is there also a full view of the ticket?

Reply:
Yeah makes sense.